app/views/overview/_networking.html

```diff
@@ -0,0 +1,59 @@
+<div ng-if="networking.services | hashSize" class="expanded-section networking-section">
+  <div class="section-title hidden-xs">Networking</div>
+  <div ng-repeat="service in networking.services" class="row">
+    <div class="col-sm-5">
+      <div class="component-label">
+        Service
+        <span class="sublabel">Internal Traffic</span>
+      </div>
+      <h3>
+        <a ng-href="{{service | navigateResourceURL}}">{{service.metadata.name}}</a>
+      </h3>
+      <!-- <span class="text-muted mar-right-xs"> -->
+      <!--   <span data-icon="" aria-hidden="true" style="font-size: 16px; vertical-align: -2px;"></span> -->
+      <!-- </span> -->
+      <span ng-repeat="portMapping in service.spec.ports | limitTo : 1">
+        {{portMapping.port}}/{{portMapping.protocol}} <span ng-if="portMapping.name">({{portMapping.name}})</span>
+        <i class="fa fa-long-arrow-right text-muted"></i>
+        {{portMapping.targetPort}}
+      </span>
+      <span ng-if="service.spec.ports.length >= 2">
+        and
+        <span class="nowrap">
+          {{service.spec.ports.length - 1}}
+          <span ng-if="service.spec.ports.length > 2">others</span>
+          <span ng-if="service.spec.ports.length === 2">other</span>
+        </span>
+      </span>
+    </div>
+    <div class="col-sm-7">
+      <div class="component-label">
+        Routes
+        <span class="sublabel">External Traffic</span>
+      </div>
+      <div ng-if="networking.routesByService[service.metadata.name] | hashSize">
+        <div ng-repeat="route in networking.routesByService[service.metadata.name] | limitTo : 2 track by (route | uid)" class="overview-routes">
+          <h3>
+            <span ng-if="route | isWebRoute">
+              <a ng-href="{{route | routeWebURL}}" target="_blank">{{route | routeLabel}}</a>
+              <i class="fa fa-external-link small" aria-hidden="true"></i>
+            </span>
+            <span ng-if="!(route | isWebRoute)">{{route | routeLabel}}</span>
+            <route-warnings route="route" service="service"></route-warnings>
```

Comment:
wondering if route warnings should be taking into account alternate backends, i know we didnt already just thinking as something for later